<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1200</width>
                <height>800</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: white;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QFrame" name="body">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_2">
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QFrame" name="controls">
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>100</height>
                                    </size>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <property name="frameShape">
                                    <enum>QFrame::StyledPanel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Raised</enum>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <property name="leftMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QFrame" name="frame">
                                            <property name="frameShape">
                                                <enum>QFrame::StyledPanel</enum>
                                            </property>
                                            <property name="frameShadow">
                                                <enum>QFrame::Raised</enum>
                                            </property>
                                        </widget>
                                    </item>
                                    <item alignment="Qt::AlignRight|Qt::AlignTop">
                                        <widget class="QFrame" name="frame_2">
                                            <property name="styleSheet">
                                                <string notr="true">color: white; border: none</string>
                                            </property>
                                            <property name="frameShape">
                                                <enum>QFrame::StyledPanel</enum>
                                            </property>
                                            <property name="frameShadow">
                                                <enum>QFrame::Raised</enum>
                                            </property>
                                            <layout class="QHBoxLayout" name="horizontalLayout_2">
                                                <property name="leftMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="topMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="rightMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="bottomMargin">
                                                    <number>0</number>
                                                </property>
                                                <item>
                                                    <widget class="QPushButton" name="to_dock">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>16</width>
                                                                <height>16</height>
                                                            </size>
                                                        </property>
                                                        <property name="styleSheet">
                                                            <string notr="true"/>
                                                        </property>
                                                        <property name="text">
                                                            <string/>
                                                        </property>
                                                        <property name="icon">
                                                            <iconset resource="icons.qrc">
                                                                <normaloff>:/icons/icons/arrow-down-left.svg</normaloff>
                                                                :/icons/icons/arrow-down-left.svg
                                                            </iconset>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QPushButton" name="minimize">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>16</width>
                                                                <height>16</height>
                                                            </size>
                                                        </property>
                                                        <property name="text">
                                                            <string/>
                                                        </property>
                                                        <property name="icon">
                                                            <iconset resource="icons.qrc">
                                                                <normaloff>:/icons/icons/minimize-2.svg</normaloff>
                                                                :/icons/icons/minimize-2.svg
                                                            </iconset>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item alignment="Qt::AlignRight|Qt::AlignTop">
                                                    <widget class="QPushButton" name="close">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>16</width>
                                                                <height>16</height>
                                                            </size>
                                                        </property>
                                                        <property name="text">
                                                            <string/>
                                                        </property>
                                                        <property name="icon">
                                                            <iconset resource="icons.qrc">
                                                                <normaloff>:/icons/icons/x.svg</normaloff>
                                                                :/icons/icons/x.svg
                                                            </iconset>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QFrame" name="content">
                                <property name="frameShape">
                                    <enum>QFrame::StyledPanel</enum>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Raised</enum>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                    <item>
                                        <widget class="QFrame" name="frame_5">
                                            <property name="frameShape">
                                                <enum>QFrame::StyledPanel</enum>
                                            </property>
                                            <property name="frameShadow">
                                                <enum>QFrame::Raised</enum>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QFrame" name="frame_3">
                                            <property name="frameShape">
                                                <enum>QFrame::StyledPanel</enum>
                                            </property>
                                            <property name="frameShadow">
                                                <enum>QFrame::Raised</enum>
                                            </property>
                                            <layout class="QVBoxLayout" name="verticalLayout_3">
                                                <item>
                                                    <widget class="QLabel" name="title">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>0</width>
                                                                <height>50</height>
                                                            </size>
                                                        </property>
                                                        <property name="maximumSize">
                                                            <size>
                                                                <width>16777215</width>
                                                                <height>50</height>
                                                            </size>
                                                        </property>
                                                        <property name="font">
                                                            <font>
                                                                <pointsize>36</pointsize>
                                                                <weight>75</weight>
                                                                <bold>true</bold>
                                                            </font>
                                                        </property>
                                                        <property name="text">
                                                            <string>Device Inventory</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QComboBox" name="user_picker">
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>0</width>
                                                                <height>30</height>
                                                            </size>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QLabel" name="error_text">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>0</width>
                                                                <height>20</height>
                                                            </size>
                                                        </property>
                                                        <property name="maximumSize">
                                                            <size>
                                                                <width>16777215</width>
                                                                <height>20</height>
                                                            </size>
                                                        </property>
                                                        <property name="styleSheet">
                                                            <string notr="true">color: red</string>
                                                        </property>
                                                        <property name="text">
                                                            <string/>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QPushButton" name="login_button">
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>0</width>
                                                                <height>30</height>
                                                            </size>
                                                        </property>
                                                        <property name="cursor">
                                                            <cursorShape>PointingHandCursor</cursorShape>
                                                        </property>
                                                        <property name="styleSheet">
                                                            <string notr="true">border-radius : 15; border: 2 px solid
                                                                black;
                                                                background-color :
                                                                #414BB2; color: white
                                                            </string>
                                                        </property>
                                                        <property name="text">
                                                            <string>LOG IN</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QPushButton" name="add_new_user_button">
                                                        <property name="cursor">
                                                            <cursorShape>PointingHandCursor</cursorShape>
                                                        </property>
                                                        <property name="styleSheet">
                                                            <string notr="true">border: none</string>
                                                        </property>
                                                        <property name="text">
                                                            <string>ADD NEW USER</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QFrame" name="frame_4">
                                            <property name="frameShape">
                                                <enum>QFrame::StyledPanel</enum>
                                            </property>
                                            <property name="frameShadow">
                                                <enum>QFrame::Raised</enum>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="footer">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>100</height>
                        </size>
                    </property>
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="icons.qrc"/>
    </resources>
    <connections/>
</ui>
